src/control/server/init/setup_spdk.sh

```diff
@@ -23,8 +23,8 @@ else
 		'/sys/class/uio/uio*/device/config'	\
 		'/sys/class/uio/uio*/device/resource*'; do
 
-		if list=$(ls $glob); then
-			echo "RUN: ls $glob | xargs -r chown -R $_TARGET_USER"
+		if list=$(ls -d $glob); then
```

Comment:
(lint) Double quote to prevent globbing and word splitting. [SC2086]

Reply:
ignore this warning because we want globbing

Reply:
If you want to ignore it you have to say so with a `# shellcheck disable=SC2086` above the line you want to ignore.  It is recommended that you do this (or fix the issue more properly/programatically as described at https://github.com/koalaman/shellcheck/wiki/SC2086) so that future readers of this code don't believe this issue to be a previously missed bug that needs fixing.